src/posts/dto/create-post.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class CreatePostDto {
+  caption: string;
+  is_hidden: string;
```

Comment:
is_hidden is a boolean. I noticed is_hidden column is string in post.entity.ts file, please modify it also

Reply:
I didn't know that because in the diagrams it was written as text not boolean

Reply:
our reliable and most updated contract is openAPI. Please check it.